### ISSUE TITLE
Preferences and menu for plugin

### ISSUE THREAD

**Issue by drama-ostrich** [Uduse/Sublime-GameMaker-Studio-Language-Bundle#5]
I started a preference menu branch in my fork so I could disable dynamic completion, which seemed to be causing noticeable lag every time I started typing a new word.

I'll open another issue for the lag from dynamic completion as soon as I have a way to demonstrate the problem. Most likely I just have some other preference set wrong, maybe I'm supposed to be ignoring some file type or directory to stop the dynamic completion from scanning the wrong files.

Anyway it seemed like it might be useful to someone if there was a menu to enable or disable parts of this plugin, and I've never worked on a sublime plugin before so I tried it out. I've started a branch here:
petroleyum@0bc31fc37da9d551461b1eef28616f7a501a8622

It isn't functional yet but it would look something like this. Let me know if you think this is useful and I'll put more work into it so it can be merged.

**Comment by uduse** [comment]
@petroleyum Yep I think that would be useful. I remember I asked a question on Sublime forum regarding the possible lag, but an admin said that he has tons of plugins enabled and it's ok to just scan everything. So I'm wondering how bad is the lag?
